test/library/standard/BigInteger/apiTest.good

```diff
@@ -0,0 +1,6 @@
+apiTest.chpl:6: warning: bigint initializers that halt are deprecated, please set the config param bigintInitThrows to 'true' to opt in to using the new initializer that throws
```

Comment:
You should be using the flag to enable throwing behavior instead of locking in these deprecation messages